packages/common/src/hooks/useNotification.ts

```diff
@@ -0,0 +1,31 @@
+import { useSnackbar } from 'notistack';
+
+interface NotificationHook {
+  error: (message: string) => () => void;
+  info: (message: string) => () => void;
+  success: (message: string) => () => void;
+  warning: (message: string) => () => void;
+}
+
+export const useNotification = (): NotificationHook => {
```

Comment:
just simplifying the usage - we may want the message removal and the closing of the snackbar.. for now this is enough

Reply:
Sorry, don't follow what you mean. Is that an enum of messages or locale message keys?